### PR TITLE
Improve docstrings

### PR DESCRIPTION
**Pull request by JoeLametta**
https://github.com/whipper-team/whipper/pull/387#issuecomment-475571334

Depends on #387.

### CODE REVIEW

**Comment by JoeLametta** [comment]
@Freso Ping. :wink:

**Comment by Freso** [comment]
> Ping. :wink:

Let’s get https://github.com/whipper-team/whipper/pull/387 done and I’ll (try and remember to) come back to this then. :)

**Comment by JoeLametta** [comment]
Rebased on develop while solving merge conflicts.

File: whipper/program/utils.py [hunk 1]
```diff
@@ -34,8 +30,9 @@ def load_device(device):
 
 def unmount_device(device):
     """
-    Unmount the given device if it is mounted, as happens with automounted
-    data tracks.
+    Unmount the given device if it is mounted.
+
+    This usueally happens with automounted data tracks.
```
Comment (ABCbum) [inline whipper/program/utils.py]: A small typo i guess 😉
```suggestion
    This usually happens with automounted data tracks.
```

**Comment by JoeLametta** [comment]
Added missing `sign-off`.

**Pull request merged**